docs/application-fixes.md

```diff
@@ -29,3 +29,31 @@ CreateDQTTRNRequest.call(application_form: application_form)
 ```
 
 Check the application is in awarded state in the UI and that the sidekiq jobs have gone through.
+
+## Rollback further information request
+
+When further information has been requested on an application and we need to ask the applicant to submit it again.
+
+**Note:** this process will lose any information provided already by the teacher, you might want to record a note that this is happening.
+
+> You can find the ID of the application from the URL: `/assessor/applications/<application-id>`
+
+In a production console:
+
+```ruby
+# find the application form and user who is performing the change
+application_form = ApplicationForm.find(...)
+user = Staff.find_by(name: ...)
+
+# set some useful variables
+assessment = application_form.assessment
+further_information_request = assessment.further_information_requests.first
+
+# delete the FI requests and reset the assessment state
+TimelineEvent.where(further_information_request:).destroy_all
```

Comment:
Could we do soft deletion on the FI request to avoid this? It probably doesn't matter in the application we're doing at the mo but changing the application timeline doesn't seem like the right thing to do generally.

Reply:
Discussed offline. This is what we have to do 'now'. If/when we expand FI to allow soft deletion or multiple requests we can update.